pyproject.toml

```diff
@@ -23,6 +23,7 @@ readme = "README.md"
 requires-python = ">=3.7"
 dependencies = [
   "sphinx>=4,<7",
+  "docutils==0.17.1", # docutils 0.18, 0.19 need a patch fix https://sourceforge.net/p/docutils/patches/195/
```

Comment:
I think we should also add a comment like "un-pin this when docutils 0.20 is released"